EasyPost/Services/BatchService.cs

```diff
@@ -103,13 +106,161 @@ public async Task<Batch> CreateAndBuy(BetaFeatures.Parameters.Batches.Create par
 
         // TODO: Add GetNextPage function when Batches are sorted newest to oldest.
 
+        /// <summary>
+        ///     Add shipments to this batch.
+        /// </summary>
+        /// <param name="parameters">Update shipment parameters.</param>
+        /// <returns>The updated Batch.</returns>
+        [CrudOperations.Update]
+        public async Task<Batch> AddShipments(string id, Dictionary<string, object> parameters)
```

Comment:
Issue: need fix the docstring

Reply:
Docstrings will be addressed in a later PR.